### ISSUE TITLE
Localnet status command fails with the latest docker compose

### ISSUE THREAD

**Issue by algolog** [algorandfoundation/algokit-cli#475]
### Subject of the issue

After upgrading to the latest Docker Desktop version 4.29.0 (145265) `algokit localnet status` command results in `Unhandled JSONDecodeError: Expecting value: line 1 column 1 (char 0)` error. The same goes for the `algokit localnet console` command.

The reason seems to be the warning message in the beginning of  `docker compose ps --format json` output: 
```
WARN[0000] /home/user/.config/algokit/sandbox/docker-compose.yml: `version` is obsolete
{"Command": ...
```
### Your environment

```
timestamp: 2024-04-15T16:10:12+00:00
AlgoKit: 2.0.2
AlgoKit Python: 3.12.2 (main, Apr  2 2024, 01:08:31) [GCC 11.4.0] (location: /home/user/.local/share/pipx/venvs/algokit)
OS: Linux-5.15.133.1-microsoft-standard-WSL2-x86_64-with-glibc2.35
docker: 26.0.0
docker compose: 2.26.1-desktop.1
git: 2.34.1
python: 3.12.2 (location: /home/user/.local/bin/python)
python3: 3.12.2 (location: /home/user/.local/bin/python3)
pipx: 1.5.0
poetry: 1.8.2
node: 20.11.1
npm: 10.2.4
```

### Steps to reproduce

1. algokit localnet reset
2. algokit localnet status

### Expected behaviour
```
# algod status
Status: Running
Port: 4001
Last round: 0
Time since last round: 0.0s
Genesis ID: dockernet-v1
Genesis hash: ze6PIDwUReOhgWJu0WaP3Mk4vA5mbjNrfEq9HHUYFLY=
Version: 3.23.1
# conduit status
Status: Running
# indexer-db status
Status: Running
# indexer status
Status: Running
Port: 8980
Last round: 0
Version: 3.4.0
```
### Actual behaviour
```
Unhandled JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

### Mitigation
Removing `version: "3"` here: 
https://github.com/algorandfoundation/algokit-cli/blob/a8874304319efc3449336a4bbb817471613d6743/src/algokit/core/sandbox.py#L387 resolves the issue. This can probably break some older compose versions. I would rather ask the docker team to report the warning in `stderr` as a better solution.

**Comment by neilcampbell** [comment]
Thanks @algolog, we'll check it out.

**Comment by neilcampbell** [comment]
I just did a bit more investigation and it seems the version field has been ignored by docker-compose for a long time now.
We have a min version check for docker-compose `2.5.0`, so we are safe to remove version and action the deprecation warning.

**Comment by neilcampbell** [comment]
Fixed in `2.0.3`
